src/xtgeo/grid3d/grid.py

```diff
@@ -1196,6 +1196,14 @@ def get_dataframe(self, activeonly=True, ijk=True, xyz=True, doubleformat=False)
     def dataframe(self, *args, **kwargs):
         return self.get_dataframe(*args, **kwargs)
 
+    def get_vtk_esg_geometry_data(self):
+        """Returns VTK style explicit structured grid (ESG) geometry data as one
+        separate vertex array and explicit connectivities for the hex elements in
+        another separate array.
+        Also tries to remove duplicate nodes, but this is WIP
+        """
```

Comment:
This is the public API part, in docstring: First line as punch line, then a line space, then give necessary explanation for the end user

Reply:
I have elaborated the docstring, also including an example of VTK usage.